ansible/main.yml

```diff
@@ -0,0 +1,110 @@
+---
+- hosts: all
+  vars:
+    app_path: /home/ubuntu/RD-WATCH
+    venv: "{{ app_path }}/venv"
+  tasks:
+  - name: checkout code
+    git:
+      repo: "https://github.com/ResonantGeoData/RD-WATCH.git"
+      dest: "{{ app_path }}"
+      force: yes
+
+  - name: Set environment variables
+    set_fact:
+      watch_environment:
+        DJANGO_CONFIGURATION:  "{{ lookup('env', 'DJANGO_CONFIGURATION') }}"
+        DJANGO_DATABASE_URL: "{{ lookup('env', 'DJANGO_DATABASE_URL') }}"
+        DJANGO_CELERY_BROKER_URL: "{{ lookup('env', 'DJANGO_CELERY_BROKER_URL') }}"
+        DJANGO_MINIO_STORAGE_ENDPOINT: "{{ lookup('env', 'DJANGO_MINIO_STORAGE_ENDPOINT') }}"
+        DJANGO_MINIO_STORAGE_ACCESS_KEY: "{{ lookup('env', 'DJANGO_MINIO_STORAGE_ACCESS_KEY') }}"
+        DJANGO_MINIO_STORAGE_SECRET_KEY: "{{ lookup('env', 'DJANGO_MINIO_STORAGE_SECRET_KEY') }}"
+        DJANGO_STORAGE_BUCKET_NAME: "{{ lookup('env', 'DJANGO_STORAGE_BUCKET_NAME') }}"
+
+  - name: install pip + python, postgres headers 
+    apt: 
+      pkg:
+        - python3-pip
+        - python3-dev
+        - libpq-dev
+      update_cache: yes
+      state: present
+    become: yes
+
+  - name: install virtualenv
+    pip:
+      name: 
+        - virtualenv
+    become: yes
+
+  - name: create virtualenv
+    shell:
+      cmd: /usr/local/bin/virtualenv {{ app_path }}/venv -p python3.8
+
+  - name: install pip requirements
+    shell: 
+      cmd: "{{ venv }}/bin/pip install -e .[dev] --find-links=https://girder.github.io/large_image_wheels"
+      chdir: "{{ app_path }}"
```

Comment:
```suggestion
  - name: install pip requirements
    pip:
      name: "{{ app_path }}"
      extra_args: "--find-links=https://girder.github.io/large_image_wheels"
      virtualenv: "{{ celery_virtualenv_path }}"
    become: true
```

Reply:
On second thought, I think you should create a new root-level `requirements.txt` in this repo, matching what we used to have here: https://github.com/ResonantGeoData/ResonantGeoData/blob/0.1.2/requirements.txt

Then, you can reference that file in this step (you'll have to rewrite it, of course).

Also, if you use https://github.com/girder/ansible-role-girder-celery , that will automagically pick up the requirements file and use it.